### DISCUSSION TITLE
VBET Metrics

### DISCUSSION THREAD

**Discussion by Cashe93** [Riverscapes/riverscapes-tools#346] [VBET] [answered]
@philipbaileynar and @wally-mac, I've attached what I have so far for these metrics. Any feedback would be great, especially when it comes to the methods. It's pretty rough, but am I including enough information for this to be useful? Do I need to add more? Add less? Let me know. 

[Metrics.xlsx](https://github.com/Riverscapes/riverscapes-tools/files/6759002/Metrics.xlsx)
[Calculation Methods.docx](https://github.com/Riverscapes/riverscapes-tools/files/6759003/Calculation.Methods.docx)

I also have some general questions that various folks might have answers to. 

1. I am having trouble with the [proportion VB by network category metric](https://github.com/Riverscapes/riverscapes-tools/discussions/108#:~:text=By%20network/riverscape%20category%20(e.g.%20perennial%2C%20intermittent%2C%20i.e.%20NHD%20FCODE)). Since the VBET outputs aren't very segmented, I haven't been able to find a way to select areas of the valley bottom that intersect with unique network types. @shelbysawyer and @lauren-herbine, it looks like there has been some discussion about how we can segment the valley bottom and I was wondering if you had access to a VBET output that has been segmented as laid out [here](https://github.com/Riverscapes/riverscapes-tools/discussions/108#discussioncomment-950491). 
2. The historic LANDFIRE raster does not have a Veg Class attribute field. Is there another attribute I should use such as the 'GROUPVEG'?
3. How does Riverscape Length differ from Channel Length? Furthermore, what do we mean by Integrated Riverscape Width mean?
4. Should there be a Land Use Intensity raster, and if so where could I find it?

**Comment by joewheaton** [comment]
Offshoot of #108

**Comment by lauren-herbine** [comment]
Hi @Cashe93- 

> I am having trouble with the proportion VB by network category metric. Since the VBET outputs aren't very segmented, I haven't been able to find a way to select areas of the valley bottom that intersect with unique network types. @shelbysawyer and @lauren-herbine, it looks like there has been some discussion about how we can segment the valley bottom and I was wondering if you had access to a VBET output that has been segmented as laid out here.

#326 is a discussion about segmentation, and I would love your input/observations/frustrations to be noted after you've been working with the current VBET segmentation. We don't actually have a VBET project with segments talked about at the end of #108. Is the issue that the network type changes (i.e. from perennial to "connector" or ephemeral or something of the like) without a corresponding segment in the VB sausage link? 

> How does Riverscape Length differ from Channel Length?

![image](https://user-images.githubusercontent.com/73319684/124617403-22e48780-de34-11eb-88de-3bd0a61bc585.png)

(this is just a quick example with rough vb margins drawn, if you're ever looking at the Flambeau, don't take these as actual truth)

I hope this helps! If you want to chat, I should be available all day

**Comment by lauren-herbine** [comment]
@Cashe93 Additionally, would you mind making your "Calculation Methods" document a vignette? It will be easy for you to update and link to a vignette hosted as a gist, and will be awesome for us to have this documentation in the same format as other things. @joewheaton lays out how to write a vignette using markdown [here](http://etal.joewheaton.org/vignettes.html). Most of what you've already written you can just copy and paste. Thanks!

**Comment by Cashe93** [comment]
Thanks for the response @lauren-herbine!  As @tyler1218hatch and myself understand it, there should be a valley bottom proportion calculated for each network type (ephemeral, intermittent, perennial, etc.). The issue I am running into is that the VBET output is not segmented so, there is no straightforward way to select portions of the valley bottom that are intersected by each network type. [Here](https://vimeo.com/573179837) is a short video that may (or may not) help clarify the issue. Also, I am in the process of converting the metrics sheet into a vignette and will send that to folks once it's completed.

**Comment by Cashe93** [comment]
Update: I have some other tasks that I need to complete this week and likely won't have much time to work on this.  @shelbysawyer, would you be able to tackle some of these metrics?

**Comment by shelbysawyer** [comment]
> Update: I have some other tasks that I need to complete this week and likely won't have much time to work on this.  @shelbysawyer, would you be able to tackle some of these metrics? 

Absolutely! Tomorrow is rig day for me and then I'm launching for my Grand Canyon trip. I'll be back Tuesday the 20th and can pick back up on tasks then. Thanks!

**Comment by Cashe93** [comment]
> Absolutely! Tomorrow is rig day for me and then I'm launching for my Grand Canyon trip. I'll be back Tuesday the 20th and can pick back up on tasks then. Thanks!

Right on! Happy floatin!

**Comment by shelbysawyer** [comment]
@Cashe93 I'm getting caught up on everything today, so I might not dive in completely on this until tomorrow. Are you around through the end of the week if I need to ping you with a few questions?

**Reply by Cashe93** [reply]
@shelbysawyer no problem! Ill be getting back into Logan tomorrow afternoon, but should have service up until then. Feel free shoot me questions anytime.

**Reply by joewheaton** [reply]
See also #346

**Comment by joewheaton** [comment, marked as answer]
Thank you all for this brain-storm. It is duplicative with:
#385
#395
